lib/v2/screens/settings/edit_name/edit_name_screen.dart

```diff
@@ -61,10 +63,18 @@ class _EditNameScreenState extends State<EditNameScreen> {
   }
 
   dynamic _onSubmitted() {
-    if (_formKeyPassword.currentState.validate()) {
-      // _editNameBloc.add(SubmitName());
-    } else {
-      // _editNameBloc.add(ActivateAutoValidate());
-    }
+    // if (_formKeyPassword.currentState.validate()) {
```

Comment:
is this commented code needed?

Reply:
Nope, I'm actually moving this forward on the Edit Name Screen branch

Reply:
I will remove it in the next commit to this PR 😄